physical_property/logP/analysis/analysis_outputs_all_submissions/StatisticsTables/statisticsLaTex/statistics.tex

```diff
@@ -0,0 +1,78 @@
+ 0T68C &                        SAMPL7\_logP\_MDPOW\_CGenFF &  1.63 [1.24, 1.98] &  1.41 [1.08, 1.76] &  -1.38 [-1.75, -1.02] &  0.54 [0.26, 0.81] &     1.26 [0.81, 1.76] &     0.52 [0.25, 0.76] &     0.46 [0.08, 0.42] \\
+ 047D4 &                                      EC\_RISM\_wet &  1.84 [1.29, 2.35] &  1.49 [1.05, 1.97] &  -1.49 [-1.97, -1.05] &  0.29 [0.05, 0.68] &     0.96 [0.37, 1.55] &     0.38 [0.08, 0.67] &     0.34 [0.02, 0.36] \\
+ 0A274 &                                 TFE-SMD-vacuum-opt &  1.96 [1.60, 2.29] &  1.76 [1.42, 2.13] &     1.76 [1.42, 2.13] &  0.44 [0.13, 0.68] &     1.04 [0.47, 1.59] &     0.41 [0.04, 0.70] &     0.31 [0.03, 0.29] \\
+ 08F3F &                          MD-EE-MCC (GAFF-TIP4P-Ew) &  2.06 [1.49, 2.57] &  1.61 [1.08, 2.15] &  -0.93 [-1.69, -0.17] &  0.03 [0.00, 0.27] &    0.47 [-0.50, 1.49] &    0.11 [-0.16, 0.38] &     0.35 [0.01, 0.38] \\
+ 0B909 &                       SAMPL7\_logP\_MDPOW\_OPLS-AA &  2.19 [1.69, 2.63] &  1.82 [1.31, 2.32] &  -1.35 [-2.03, -0.58] &  0.28 [0.07, 0.58] &     1.47 [0.62, 2.50] &     0.36 [0.08, 0.62] &     0.38 [0.04, 0.37] \\
+ 08DA5 &                   TFE-prediction-solvation-b3lypd3 &  2.19 [1.76, 2.57] &  1.98 [1.58, 2.38] &     1.98 [1.58, 2.38] &  0.40 [0.09, 0.67] &     1.06 [0.44, 1.62] &     0.45 [0.10, 0.71] &    0.04 [-0.00, 0.06] \\
+ 6Q594 &                     SAMPL7\_logP\_MDPOW\_LigParGen &  2.28 [1.80, 2.72] &  1.95 [1.46, 2.45] &    0.35 [-0.61, 1.29] &  0.07 [0.00, 0.38] &    0.83 [-0.50, 2.26] &    0.19 [-0.14, 0.50] &     0.38 [0.04, 0.36] \\
+ 085FC &                                TFE-SMD-solvent-opt &  2.39 [1.98, 2.77] &  2.19 [1.80, 2.58] &     2.19 [1.80, 2.58] &  0.40 [0.10, 0.67] &     1.09 [0.46, 1.69] &     0.42 [0.10, 0.69] &     0.20 [0.02, 0.21] \\
+ 0636A &                                  TFE-NHLBI-TZVP-QM &  2.72 [1.93, 3.43] &  2.17 [1.51, 2.90] &     2.08 [1.38, 2.84] &  0.52 [0.19, 0.78] &  -1.16 [-1.65, -0.58] &  -0.51 [-0.79, -0.19] &    0.02 [-0.00, 0.07] \\
+ 2DF02 &                         Ensemble prediction of TFE &  2.73 [2.27, 3.15] &  2.54 [2.13, 2.96] &     2.54 [2.13, 2.96] &  0.33 [0.05, 0.64] &  -0.30 [-0.49, -0.10] &  -0.35 [-0.60, -0.02] &  -0.00 [-0.00, -0.00] \\
+ 004AF &                         Ensemble prediction of TFE &  3.29 [2.88, 3.68] &  3.16 [2.77, 3.55] &     3.16 [2.77, 3.55] &  0.39 [0.05, 0.73] &  -0.25 [-0.40, -0.09] &  -0.46 [-0.71, -0.13] &  -0.00 [-0.00, -0.00] \\
+ 09F0E &             RayLogP三\_QSPR\_Mordred2D\_TPOT-AutoML &  3.64 [3.01, 4.24] &  3.36 [2.79, 3.98] &     3.36 [2.79, 3.98] &  0.39 [0.10, 0.70] &  -0.72 [-1.11, -0.33] &  -0.37 [-0.65, -0.03] &  -0.00 [-0.00, -0.00] \\
+ 03F9F &                                    TFE-NHLBI-NN-IN &  3.97 [3.57, 4.35] &  3.85 [3.45, 4.25] &     3.85 [3.45, 4.25] &  0.00 [0.00, 0.15] &    0.02 [-0.30, 0.35] &    0.02 [-0.22, 0.26] &    0.01 [-0.00, 0.02] \\
+\end{longtable}
```

Comment:
Would be better if this used submission IDs rather than hexadecimal codes.

Reply:
This has been updated to list the method names and submission file names